_episodes/01-rebase.md

```diff
@@ -47,9 +50,10 @@ $ git rebase master
 ### Advantages and disadvantages
 
 - `git rebase` makes "merges" producing a linear history.
+- `git merge` resolves all conflicts in a single commit, with `git rebase` each commit being rebased needs 
+  conflict resolution.
```

Comment:
"each commit may need" - only the conflicting commits need conflict resolution, not necessarily each commit.

Reply:
indeed, i'll fix it, thanks